resources/js/requests/components/RequestsListing.vue

```diff
@@ -332,7 +334,7 @@ export default {
       let htmlString = '';
       for (const task of value) {
         htmlString += `
-          <div>
+          <div class="text-truncate">
```

Comment:
Amazing thanks for the suggestion I added the code to the SavedSearchController [PR #410](https://github.com/ProcessMaker/package-savedsearch/pull/410).

**But this change is needed**, when a request has two or more active tasks the `active_tasks` column it will show two or more lines, like the following:

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/019faf5e-9f4f-4fa1-a7b1-35a356f06546)

As you could see in the previous image, the ellipsis does not appear becasue of the use of `<div>` to render multiple lines of tasks.

This line of code adds a `text-trucate` to each active task `div` of the request to add the ellipsis like:

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/cdfda16e-f05e-4943-8fb1-bc50a7d44abc)